ipaserver/install/ca.py

```diff
@@ -177,14 +177,14 @@ def install_check(standalone, replica_config, options):
     if standalone:
         dirname = dsinstance.config_dirname(
             installutils.realm_to_serverid(realm_name))
-        cadb = certs.CertDB(realm_name, subject_base=options._subject_base)
+        cadb = certs.CertDB(realm_name, paths.HTTPD_ALIAS_DIR,
+                            subject_base=options._subject_base)
```

Comment:
The same NSS database as below in `install()` should be used here, i.e. `paths.PKI_TOMCAT_ALIAS_DIR`.